docs/source/docs/registry.md

```diff
@@ -106,3 +108,18 @@ To register a custom backbone class, the steps are as follows
 Please check the self-contained
 [example script](https://github.com/DeepRegNet/DeepReg/blob/main/examples/custom_backbone.py)
 for further details.
+
+### Custom Loss
+
+To register a custom image loss class, the steps are as follows
+
+1. Subclass the `tf.keras.losses.Loss` and implement a custom backbone class.
+2. Import `REGISTRY` and use the decorator `@REGISTRY.register_loss` to register the
+   custom class.
+3. Use the registered name in the config for using the registered backbone.
```

Comment:
be specific: "registered name for the new custom loss"

Reply:
and its parameter as appropriate - should we use an example with parameter? too complicated

Reply:
good point, for parameterized loss, I will do another separate example